Swabbr.Api/Authentication/JwtConfiguration.cs

```diff
@@ -23,6 +23,11 @@ public class JwtConfiguration
         /// <summary>
         ///     Token expire time in minutes.
         /// </summary>
-        public uint TokenValidity { get; set; }
+        public int TokenValidity { get; set; }
```

Comment:
It seems that it's actual usage requires this value to be a `Double`. You can declare it as a `Double` here and avoid casting altogether.

Reply:
Uint is non-cls compliant. Setting a uint for a value we will always configure to be positive once during deployment / configuration doesn't bring any added benefits. It only introduces unnecessary complexity when handling these configured uint values as a regular int is almost always expected.

Using a double could be an option but since this parameter indicates whole minutes a double wouldn't match the intended purpose for this method. The only real reason a double would be suggested here is because the `TimeSpan.FromMinutes(double minutes)` expects a double. This could have been any other method.